lib/definitions/constants.js

```diff
@@ -1,3 +1,10 @@
 export const HOME_URL = 'https://github.com/semantic-release/semantic-release';
 
 export const RELEASE_NAME = 'GitLab release';
+
+export const RETRY_CONFIGURATION = {
+    retry: {
```

Comment:
We could try to calculate the `retryAfter` property into optimizing the calculation for the next try.

Reply:
I think `got` does that already, see https://github.com/sindresorhus/got/blob/cc346b79561b13e3a266b22b7f612fca5a45eb1e/source/core/index.ts#L357

Reply:
Yes indeed. I assumed we are overriding the `retry` options when setting the `apiOptions`. I was wrong. Thanks for properly looking into it. I will close this PR.
What is still confusing though are the reported `429` responses of some users...